man/hwdb-usb-device.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <sd-hwdb.h>
+
+int print_usb_properties(uint16_t vid, uint16_t pid) {
+  char match[15];
+  sd_hwdb *hwdb;
+  const char *key, *value;
+  int r;
+
+   /* Search fields matching by USB vendor/product IDs */
```

Comment:
indentation slightly off

Reply:
Fixed now. I changed the comment a bit too.